jobs/sql/extractions/functions/temp_program.sql

```diff
@@ -0,0 +1,355 @@
+-- You should uncomment this line to check syntax in IDE.  Liquibase handles this internally.
```

Comment:
Might be good add a 2-line summary of the uses of these procedures?  (Looks like utility functions to create temp tables for a variety of sets of encounters and obs that happen within a specific program... seems cool and helpful...)